src/main/resources/avro/alleleAnnotations.avdl

```diff
@@ -107,6 +107,22 @@ record VariantAnnotationSet {
 }
 
 /**
+A HGVSAnnotation record holds Human Genome Variation Society descriptions
+of the variant allele with respect to a transcript.
+*/
+record HGVSAnnotation {
+
+  /** HGVS formatted annotation at genomic level  */
```

Comment:
I'm afraid I don't find these comments hugely enlightening. I already know that it's a "HGVS formatted annotation" since it's held in this struct. Would it be possible to get a slightly higher-level description? Perhaps some links to the HGVS details?